src/main/java/org/openstreetmap/josm/plugins/pt_assistant/gui/PTAssistantSnackbar.java

```diff
@@ -0,0 +1,94 @@
+// License: GPL. For details, see LICENSE file.
+
+package org.openstreetmap.josm.plugins.pt_assistant.gui;
+
+import static org.openstreetmap.josm.tools.I18n.tr;
+
+import java.awt.Color;
+import java.awt.Cursor;
+import java.awt.Font;
+import java.awt.GridBagLayout;
+import java.util.Timer;
+import java.util.TimerTask;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.border.CompoundBorder;
+import javax.swing.border.EmptyBorder;
+import javax.swing.border.EtchedBorder;
+
+import org.openstreetmap.josm.gui.MainApplication;
+import org.openstreetmap.josm.gui.widgets.JMultilineLabel;
+import org.openstreetmap.josm.tools.Logging;
+import org.openstreetmap.josm.tools.GBC;
+import org.openstreetmap.josm.gui.MapFrame;
+import org.openstreetmap.josm.tools.ImageProvider;
+
+/**
+ * Snackbars (or useful messages) that are displayed on the bottom of the map
+ *
+ * @author sudhanshu2
+ */
+public class PTAssistantSnackbar extends JPanel {
+
+    /**
+     * Creates a snackbar
+     *
+     * @param message the plain text string to be displayed
+     * @param seconds time the message should be displayed for, set it to -1 for infinite (persist message)
+     */
+    public PTAssistantSnackbar(String message, long seconds) {
+        Font font = getFont().deriveFont(Font.PLAIN, 14.0f);
+        JMultilineLabel snackBarLabel = new JMultilineLabel(tr(message));
```

Comment:
This won't work - remove the tr call

tr is doing some magic - it only works with fixed strings as first parameter:

```java
// good
tr("Some text")
// bad
tr("Somte" + "text")
// bad
tr(someVariable)
```